src/userInterface/util/enums/MergeModes.ts

```diff
@@ -0,0 +1,33 @@
+export enum MergeModes {
+    diff = 'diff',
+    reload = 'reload',
+    save = 'save'
+}
+
+export interface MergeText {
+    btnLeft: string;
+    btnRight: string;
+    txtLeft: string;
+    txtRight: string;
+}
+
+export const MergeModesMap: Map<string, MergeText> = new Map([
```

Comment:
meka this a constant fil. no need to create map for this. you can access const values like mergeModes[enumString]